sys/include/usb/usbus.h

```diff
@@ -281,7 +281,7 @@ typedef struct usbus_interface_alt {
 typedef struct usbus_interface {
     struct usbus_interface *next;       /**< Next interface (set by USBUS during
                                              registration)                     */
-    usbus_hdr_gen_t *hdr_gen;           /**< Optional additional header
+    usbus_descr_gen_t *descr_gen;       /**< Optional additional descriptor
                                              generators                        */
```

Comment:
generator or generators ? Must be align with line 272 of the same file.

Reply:
Something something linked list of generators, clarified and fixed all occurences.